flow/scripts/global_route.tcl

```diff
@@ -13,7 +13,11 @@ proc global_route_helper { } {
       -congestion_report_file $::global_route_congestion_report] \
       $::env(GLOBAL_ROUTE_ARGS)]
 
-    log_cmd global_route {*}$all_args
+    if { [env_var_exists_and_non_empty ENABLE_RESISTANCE_AWARE] } {
+      log_cmd global_route {*}$all_args -resistance_aware
+    } else {
+      log_cmd global_route {*}$all_args
+    }
```

Comment:
We have some helpers for adding flags, see repair_timing_helper for example.  I would expect something like
```
  append_env_var all_args ENABLE_RESISTANCE_AWARE -resistance_aware 0
```